### PR TITLE
Refactor to async

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: test/BuildingRegistry.Tests/BackOffice/Lambda/Building/BuildingLambdaHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+            var buildingPersistentLocalId = Fixture.Create<BuildingPersistentLocalId>();
+
+            PlanBuilding(buildingPersistentLocalId);
+
+            var sut = new FakeBuildingLambdaHandler(
+                Container.Resolve<IConfiguration>(),
+                new FakeRetryPolicy(),
+                Container.Resolve<IBuildings>(),
+                ticketing.Object,
+                MockExceptionIdempotentCommandHandler(() => new IdempotencyException(string.Empty)).Object);
```
Comment (pgallik) [inline test/BuildingRegistry.Tests/BackOffice/Lambda/Building/BuildingLambdaHandlerTests.cs:108]: Setup the test with a mock exception idempotent handler without the exception.

File: src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Requests/BackOfficeNotRealizeBuildingRequest.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+namespace BuildingRegistry.Api.BackOffice.Abstractions.Building.Requests
+{
+    using System.Runtime.Serialization;
+
+    [DataContract(Name = "NietRealiseerGebouw", Namespace = "")]
```
Comment (ArneD) [inline src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Requests/BackOfficeNotRealizeBuildingRequest.cs:5]: remove datacontract

File: src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Requests/BackOfficePlaceBuildingUnderConstructionRequest.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+namespace BuildingRegistry.Api.BackOffice.Abstractions.Building.Requests
+{
+    using System.Runtime.Serialization;
+
+    [DataContract(Name = "GebouwInAanbouw", Namespace = "")]
```
Comment (ArneD) [inline src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Requests/BackOfficePlaceBuildingUnderConstructionRequest.cs:5]: remove `DataContract`

File: src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Requests/BackOfficeRealizeBuildingRequest.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+namespace BuildingRegistry.Api.BackOffice.Abstractions.Building.Requests
+{
+    using System.Runtime.Serialization;
+
+    [DataContract(Name = "RealiseerGebouw", Namespace = "")]
```
Comment (ArneD) [inline src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Requests/BackOfficeRealizeBuildingRequest.cs:5]: remove `DataContract`

File: src/BuildingRegistry.Api.BackOffice.Abstractions/BuildingUnit/Requests/BackOfficeNotRealizeBuildingUnitRequest.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+namespace BuildingRegistry.Api.BackOffice.Abstractions.BuildingUnit.Requests
+{
+    using System.Runtime.Serialization;
+
+    [DataContract(Name = "NietRealiseerGebouweenheid", Namespace = "")]
```
Comment (ArneD) [inline src/BuildingRegistry.Api.BackOffice.Abstractions/BuildingUnit/Requests/BackOfficeNotRealizeBuildingUnitRequest.cs:5]: remove `DataContract`

File: src/BuildingRegistry.Api.BackOffice.Abstractions/BuildingUnit/Requests/BackOfficeRealizeBuildingUnitRequest.cs [hunk 6]
```diff
@@ -0,0 +1,10 @@
+namespace BuildingRegistry.Api.BackOffice.Abstractions.BuildingUnit.Requests
+{
+    using System.Runtime.Serialization;
+
+    [DataContract(Name = "RealiseerGebouweenheid", Namespace = "")]
```
Comment (ArneD) [inline src/BuildingRegistry.Api.BackOffice.Abstractions/BuildingUnit/Requests/BackOfficeRealizeBuildingUnitRequest.cs:5]: remove `DataContract`

**Review by ArneD — CHANGES_REQUESTED** [review]

**Pull request merged**